File: sdks/thunkmetrc-ts/wrapper/src/models/Facility.ts

import type { FacilityFacilityType } from './FacilityFacilityType';
import type { FacilityLicense } from './FacilityLicense';
export interface Facility {
    Alias?: string;
    AllowTransferOfOnHoldPackages?: boolean;
    AllowTransferOfOnRecallPackages?: boolean;
    CredentialedDate?: string;
    DisplayName?: string;
    Email?: string;
    FacilityId?: number;
    FacilityType?: FacilityFacilityType;
    HireDate?: string;
    IsFinancialContact?: boolean;
    IsManager?: boolean;
    IsOwner?: boolean;
    License?: FacilityLicense;
    Name?: string;
    Occupations?: any[];
    SupportActivationDate?: string;
    SupportExpirationDate?: string;
}

File: sdks/thunkmetrc-py/wrapper/src/thunkmetrc/wrapper/models/facility.py
from typing import TypedDict, List, Optional, Any, Generic, TypeVar, TYPE_CHECKING
from typing_extensions import TypedDict

if TYPE_CHECKING:
    from . import *

class Facility(TypedDict, total=False):
    Alias: str
    AllowTransferOfOnHoldPackages: bool
    AllowTransferOfOnRecallPackages: bool
    CredentialedDate: str
    DisplayName: str
    Email: str
    FacilityId: int
    FacilityType: 'FacilityFacilityType'
    HireDate: str
    IsFinancialContact: bool
    IsManager: bool
    IsOwner: bool
    License: 'FacilityLicense'
    Name: str
    Occupations: List[Any]
    SupportActivationDate: str
    SupportExpirationDate: str
